src/cluster/k8sClientHandler.go

```diff
@@ -475,3 +475,25 @@ func GetNodesFromK8sClient() (*v1.NodeList, error) {
 	}
 	return nodeList, nil
 }
+
+func GetKubearmorRelayURL() string {
+	var namespace string
+	client := ConnectK8sClient()
+	if client == nil {
+		return ""
+	}
```

Comment:
Is there no global client which we can access in the engine?  Do we create a new client every time we make k8s API calls in the engine?

Reply:
yes, right now there is no global client present.

Reply:
Let's create a note for it, might  need to handle it in future.